Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -2440,15 +2440,14 @@ void MolDraw2D::extractMolNotes(const ROMol &mol) {
   }
 
   if (!note.empty()) {
-    auto note_rect = calcAnnotationPosition(mol, note);
-    if (note_rect.width_ < 0.0) {
+    AnnotationType annot;
+    annot.text_ = note;
+    annot.align_ = TextAlignType::START;
+    annot.scaleText_ = false;
+    calcAnnotationPosition(mol, annot);
+    if (annot.rect_.width_ < 0.0) {
       cerr << "Couldn't find good place for molecule note " << note << endl;
```

Comment:
again, not part of this PR, but:
```suggestion
      BOOST_LOG(rdWarningLog) << "Couldn't find good place for molecule note " << note << std::endl;
```